package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -35,6 +26,11 @@ def _is_json_serializable(obj: Any):
 class SQLiteStore(BaseSessionStore):
     """Stores the session data on the sqlite db."""
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        session_class = create_db_engine(self.location)
+        self._db_session = session_class
```

Comment:
```suggestion
        self._db_session = create_db_engine(self.location)
```
Super minor style improvement...